index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Good job @dagmawiii03, but I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.
- [ ] Kingly, make sure you don't forget to highlight the effort of the student to encourage him, you could say something like `Good job for using...`,`Good job @dagmawiii03...` ... 👍

Reply:
Done...by editing the previous comments